performance/get_time.js

```diff
@@ -1,15 +1,16 @@
-const path = require('path');
+const { pathToFileURL } = require('url');
+const process = require('process');
 const { VASTClient } = require('../dist/vast-client-node.min.js');
 
-const url = path.join('file://', __dirname, '..', 'spec', 'samples', 'inline-linear.xml');
+const url = pathToFileURL('spec/samples/inline-linear.xml').toString();
 
 const VastClient = new VASTClient();
 
-const timeBefore = Date.now();
-
+const timeBefore = process.hrtime();
 VastClient.get(url)
   .then(() => {
-    console.log(Date.now() - timeBefore);
+    const res = process.hrtime(timeBefore);
+    console.log((res[0] * 1e3 + res[1] * 1e-6));
```

Comment:
I think it should be `1e6` not `1e-6`? (same in the other file)

```
1e6 === 1000000
1e-6 === 0.000001
```
Maybe more readable to just write 1000 / 1000000 instead :)
Or even better, since recent versions of node you can write integers with `_` separators for readability:
`1_000`
`1_000_000`

Reply:
`res[1]` is in nanoseconds so I'm multiplying it by 1e-6, I could also divide it by 1e6, but it is correct as it is. I can change it to be more explicit

Reply:
Ah well, the node docs [suck](https://nodejs.org/api/process.html#process_process_hrtime_time) because they look like the `arr[1]` is a small value while in reality it's huge.

And now I see that you use `ms` while they use `ns` in output. All good 👍